pkg/kubectl/cmd/expose.go

```diff
@@ -37,6 +37,9 @@ re-use the labels from the resource it exposes.`
 	expose_example = `// Creates a service for a replicated nginx, which serves on port 80 and connects to the containers on port 8000.
 $ kubectl expose rc nginx --port=80 --target-port=8000
 
+// Creates a service for a replicated nginx from nginx-controller.yaml, which serves on port 80 and connects to the containers on port 8000.
```

Comment:
Like other kubectl reviews, change `//` to `#` and claify that the file here is used only to get the identity of the resource.